MonoGame.Framework/Graphics/SpriteBatchItem.cs

```diff
@@ -19,70 +19,77 @@ public SpriteBatchItem ()
             vertexBR = new VertexPositionColorTexture();            
 		}
 		
-		public void Set ( float x, float y, float w, float h, Color color, Vector2 texCoordTL, Vector2 texCoordBR )
+		public void Set ( float x, float y, float w, float h, Color color, Vector2 texCoordTL, Vector2 texCoordBR, float depth )
 		{
 			vertexTL.Position.X = x;
```

Comment:
@nkast - I wonder if it is better to store the `Set` data in the `SpriteBatchItem` in a more efficient format making this structure much smaller.... then building the `VertexPositionColorTexture` at VB packing time.